.travis.yml

```diff
@@ -1,3 +1,4 @@
+---
```

Comment:
> 9.1.4. Explicit Documents

> An explicit document begins with an explicit directives end marker line but no directives. Since the existence of the document is indicated by this marker, the document itself may be completely empty.